packages/src/@terra-money/wallet-controller/modules/wallet-plugin/types.ts

```diff
@@ -16,6 +16,7 @@ export interface WalletPluginSession {
   terraAddress: string | null;
   connect: () => Promise<void>;
   disconnect: () => Promise<void>;
+  getMetadata: () => { [key: string]: any };
```

Comment:
And here I'll set getMetadata() as an optional parameter
```suggestion
  getMetadata?: () => { [key: string]: any };
```
If you think it's better, you can leave this as a mandatory parameter and remove the check on controller.ts